src/amo/containers/Home.js

```diff
@@ -6,6 +7,22 @@ import translate from 'core/i18n/translate';
 
 import 'amo/css/Home.scss';
 
+const CategoryLink = ({ name, slug, text, type }) => (
```

Comment:
This feels really specific to me to put in its own file and they aren't exported right now.

I just think this is nicer than doing a helper like `this.extensionLink({ name, slug, text })`.